Gui/src/main/java/mezz/jei/gui/config/ClientConfig.java

```diff
@@ -42,6 +43,11 @@ public ClientConfig(IConfigSchemaBuilder schema) {
 			false,
 			"Enable cheating items into the hotbar by using the shift+number keys."
 		);
+		addBookmarksToFront = advanced.addBoolean(
+			"AddBookmarksToFrontEnabled",
+			false,
```

Comment:
```suggestion
			true,
```
Please keep the existing behavior here, users can configure it but I do not want to cause confusion by changing it by default.